scripts/RobotFrameworkSetup.iss

```diff
@@ -34,6 +34,7 @@
 #define MyAppVersion "RobotFramework AIO " + RobotFrameworkVersion + " (Installer " + SETUPVersion + ")"
 #define MyAppFileName "RobotFramework_AIO_setup_" + RobotFrameworkVersion
 #define MyAppPublisher "Robert Bosch GmbH"
+#include '..\Output\Include\install_projects.iss';
```

Comment:
Good idea to load this from outside.
Please consider, that a manual build need to be possible to...

Reply:
Hi Thomas,

I tested it manual in my PC. The manual build was not effect by this setup.
When user run ./build the precompiler.bat (which is already called in ./build) will read the json config and generate '..\Output\Include\install_projects.iss'.
There are no more action for manual build now. For gitlab build, we should input the path of internal json config to ./build . I will push these commits into gitlab.

Reply:
O.K. very good!